pkg/kotsutil/troubleshoot.go

```diff
@@ -50,3 +51,26 @@ func LoadTSKindsFromPath(dir string) (*troubleshootloader.TroubleshootKinds, err
 	}
 	return tsKinds, nil
 }
+
+func Dedup[T any](objs []T) []T {
```

Comment:
can you move this to the supportbundle package? this is not a generic dedup package as it relies on marshalling the object and this will not work for many types

Reply:
what types don't work with json.Marshal?

Reply:
lower case fields of structs

Reply:
if it were generic it would use reflection